src/build.d

```diff
@@ -131,6 +131,14 @@ Command-line parameters
     if (res.helpWanted)
         return showHelp;
 
+    // Since we're ultimately outputting to a TTY, force colored output
+    // A more proper solution would be to redirect DMD's output to this script's
+    // output using `std.process`', but it's more involved and the following
+    // "just works"
```

Comment:
Unfortunately it only "just works" on Posix where color codes are part of the output. 
Worse, since https://github.com/dlang/dmd/pull/12541 got merged, this is what you now get on Windows:
```
-----------------------------------------------------------                                          
←[1msrc\dmd\mars.d(19): ←[1;31mError: ←[mno identifier for declarator `←[0;36m←[m←[1merror←[0;36m←[m`
```

Reply:
@dkorpel which terminal are you using on Windows? I think even the built-in one supports ANSI escape codes these days.

Reply:
> I think even the built-in one supports ANSI escape codes these days

I checked and indeed they're supported, but require  to be enabled explicitly:
https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences?redirectedfrom=MSDN